docs/src/assets/risk.csv

```diff
@@ -0,0 +1,1001 @@
+id,Risk

```

Comment:
can you please add a source information for this file to readme (I assume you made up this file yourself?)